spec/recurly/subscription_spec.rb

```diff
@@ -69,6 +69,28 @@
       subscription = Subscription.new attributes
       subscription.to_xml.must_equal get_raw_xml("subscriptions/serialize-with-bulk.xml")
     end
+
+    describe 'when plan is ramp pricing' do
+      before do
+        attributes.merge!(
+          ramp_intervals: [
+            SubscriptionRampInterval.new(
+              starting_billing_cycle: 1,
+              unit_amount_in_cents: 1000
+            ),
+            SubscriptionRampInterval.new(
+              starting_billing_cycle: 2,
+              unit_amount_in_cents: 2000
+            )
+          ]
+        )
+      end
+
+      it 'properly serializes' do
+        subscription = Subscription.new attributes
+        subscription.to_xml.must_equal get_raw_xml('subscriptions/serialize-with-ramps.xml')
```

Comment:
Now I see why its formatted like that.

Reply:
```ruby
subscription.to_xml.must_equal get_raw_xml('subscriptions/serialize-with-ramps.xml').split("\n").map(&:strip).join
```

if we're doing this sort of equality checks regularly, then perhaps add a 2nd optional parameter to get_raw_xml for `compact: true` and encapsulate the `.split("\n").map(&:strip).join` bit.